package.json

```diff
@@ -47,5 +48,9 @@
   },
   "engines": {
     "node": "6.* || 8.* || >= 10.*"
+  },
+  "dependencies": {
+    "g": "^2.0.1",
```

Comment:
What part of the app is using this npm package? I don't think this is something we need.